### PR TITLE
Hsp90 woodhead urokinase scyt dehyd

### PR DESCRIPTION
**Pull request by vgapsys**
## Input submission checklist

* [x] created a directory for each system based on the [template directory][templates]
* [x] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [x] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [x] added a PDB file with the protein named `protein.pdb`
* [x] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [x] did not re-protonate the residues, but left protonation state of the protein unchanged
* [x] kept cystallographic waters and metals in the PDB file
* [x] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [x] copied over the ligands SDF file to a file named `ligands.sdf` and removed any duplicate binding modes or protonation states, keeping the more favorable state

## Summary of changes
Detailed summary in each folder

## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: industry_benchmarks/input_structures/prepared_structures/water_set/hsp90_woodhead/PREPARATION_DEATILS.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+protein:
+ - used maestro protein preparation to cap C-term (N-term was already capped)
```
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/water_set/hsp90_woodhead/PREPARATION_DEATILS.md:3]: Could you add the maestro version here?

File: industry_benchmarks/input_structures/prepared_structures/water_set/urokinase/PREPARATION_DEATILS.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+protein:
+ - chainA: cap N- and C-termini
```
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/water_set/urokinase/PREPARATION_DEATILS.md:3]: If the capping was done with maestro, could you add the version number here?

**Review by hannahbaumann — CHANGES_REQUESTED** [review]
Thanks @vgapsys , looks good to me! Just the two small comments about the maestro version number for when we write the paper.

File: industry_benchmarks/input_structures/prepared_structures/water_set/hsp90_woodhead/PREPARATION_DEATILS.md (hunk 1 above)
Comment (vgapsys) [reply industry_benchmarks/input_structures/prepared_structures/water_set/hsp90_woodhead/PREPARATION_DEATILS.md]: done

File: industry_benchmarks/input_structures/prepared_structures/water_set/urokinase/PREPARATION_DEATILS.md (hunk 2 above)
Comment (vgapsys) [reply industry_benchmarks/input_structures/prepared_structures/water_set/urokinase/PREPARATION_DEATILS.md]: done

**Review by hannahbaumann — APPROVED** [review]
Thanks @vgapsys , looks good to me!

**Review by IAlibay — APPROVED** [review]
Thanks @vgapsys !

**Pull request merged**